presto-docs/src/main/sphinx/installation/jmx.rst

```diff
@@ -0,0 +1,66 @@
+===================
+Monitoring with JMX
+===================
+
+Presto exposes a large number of different metrics via the Java Management
+Extensions (JMX). JConsole, supplied with the JDK, VisualVM, and many other
+tools can be used to access the metrics in a client application. Many monitoring
+solutions support JMX.
+
+You can configure the ports used by JMX in ``jvm.config`` as described in
+:ref:`config_properties`.
+
+Alternatively you can use the :doc:`/connector/jmx` and query the metrics using
+SQL.
+
+Many of these JMX metrics are a complex metric object such as a ``CounterStat``
+that has a collection of related metrics. For example, ``InputPositions`` has
+``InputPositions.TotalCount``, ``InputPositions.OneMinute.Count``, and so on.
+
+A small subset of the available metrics are described below.
+
+JVM
+---
+
+* Heap size: ``java.lang:type=Memory:HeapMemoryUsage.used``
+* Thread count: ``java.lang:type=Threading:ThreadCount``
+
+Presto Cluster and Nodes
+------------------------
+
+* Active nodes:
+``presto.failureDetector:name=HeartbeatFailureDetector:ActiveCount``
```

Comment:
These need to be indented:
```
* Active nodes:
  ``presto.failureDetector:name=HeartbeatFailureDetector:ActiveCount``
```
It's failing the build:
```
/Users/dphillips/src/presto/presto-docs/src/main/sphinx/installation/jmx.rst:32:Bullet list ends without a blank line; unexpected unindent.
```

Reply:
thanks for the fix